### PR TITLE
[Catalog] Use new MDCColorScheming APIs for theming the catalog

### PR DESCRIPTION
**Pull request by jverkoey**
Updates the AppTheme to make use of the MDCColorScheming APIs.

Pivotal story: https://www.pivotaltracker.com/story/show/156522700

simulator screen shot - iphone 8 plus - 2018-04-10 at 13 17 38

simulator screen shot - iphone 8 plus - 2018-04-10 at 13 17 47

### CODE REVIEW

**Comment by jverkoey** [comment]
Blocked by https://github.com/material-components/material-components-ios/pull/3283

**Comment by jverkoey** [comment]
Ready for review.

File: catalog/MaterialCatalog/MDCCatalogTiles.m [hunk 1]
```diff
@@ -36,7 +36,7 @@
 #pragma clang diagnostic ignored "-Wassign-enum"
 #pragma clang diagnostic ignored "-Wconversion"
 
-void MDCCatalogDrawMDCLogoDark(CGRect frame, id<MDCColorScheme> colorScheme) {
+void MDCCatalogDrawMDCLogoDark(CGRect frame, id<MDCColorScheming> colorScheme) {
```
Comment (romoore) [inline catalog/MaterialCatalog/MDCCatalogTiles.m:39]: Is this even used? I don't see any references in the method body

**Review by romoore — APPROVED** [review]
Please add 1 or 2 before/after screenshots. It looks like the tiles won't be themed.

File: catalog/MaterialCatalog/MDCCatalogTiles.m (hunk 1 above)
Comment (jverkoey) [reply catalog/MaterialCatalog/MDCCatalogTiles.m:39]: Nope. But will remove in a separate PR.

**Comment by jverkoey** [comment]
Added screenshots.

**Pull request merged**